build.xml

```diff
@@ -134,6 +134,8 @@
     <property name="chronicle-wire.version" value="2.20.117" />
     <property name="chronicle-threads.version" value="2.20.111" />
 
+    <property name="dtest-api.version" value="0.0.12" />
```

Comment:
Since this won't build without .13 anyway, should this be 0.0.13?

Reply:
sure